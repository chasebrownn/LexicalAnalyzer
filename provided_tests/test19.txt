RxDoJ ((U)|(d)).((F).(4)) #
" UF4 UF4 dF4 UF4UF4 dF4dF4 " 