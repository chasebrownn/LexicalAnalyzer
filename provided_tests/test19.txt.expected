RxDoJ , "UF4"
RxDoJ , "UF4"
RxDoJ , "dF4"
RxDoJ , "UF4"
RxDoJ , "UF4"
RxDoJ , "dF4"
RxDoJ , "dF4"
